w-books/src/config/routes.js

```diff
@@ -2,17 +2,51 @@ import React from 'react';
 import {
   BrowserRouter as Router,
   Route,
-  Link
+  Link,
+  Redirect
 } from 'react-router-dom'
 import Home from '../App/screens/Home/index.js'
-import BookDetail from '../App/screens/BookDetail';
+import BookDetail from '../App/screens/BookDetail/index.js';
+import Login from '../App/screens/Login/index.js';
+
+const isAuthenticated = () => {
+  return localStorage.getItem('access_token') !== null;
+}
+
+const PrivateRoute = ({ component: Component, ...rest }) => (
+  <Route {...rest} render={props => (
+    isAuthenticated() ? (
+      <Component {...props}/>
+    ) : (
+      <Redirect to={{
+        pathname: '/login',
+        state: { from: props.location }
+      }}/>
+    )
+  )}/>
+)
+
+const PublicRoute = ({ component: Component, ...rest }) => (
```

Comment:
extract this functions ( `PublicRoute` and `PrivateRoute` ) into a external file with this names each.